.goreleaser.yml

```diff
@@ -9,34 +10,33 @@ brew:
     owner: gocardless
     name: homebrew-taps
   commit_author:
-    name: Harry Maclean
-    email: harrymaclean@gocardless.com
+    name: GoCardless Engineering (goreleaser)
+    email: engineering@gocardless.com
   install: bin.install "draupnir-client"
   description: Client for the draupnir database service
   folder: Formula
   test: system "draupnir-client help"
 builds:
-- 
-  main: ./main.go
-  binary: draupnir
-  goos:
-  - linux
-  - darwin
-  main: .
-  ldflags: -s -w -X github.com/gocardless/draupnir/version.Version={{.Version}} -X main.commit={{.Commit}}
--
-  binary: draupnir-client
-  main: ./cli/main.go
-  ldflags: -s -w -X github.com/gocardless/draupnir/version.Version={{.Version}} -X main.commit={{.Commit}}
-  goos:
-  - linux
-  - darwin
+  -
+    main: ./main.go
+    binary: draupnir
+    goos:
+      - linux
+      - darwin
+    ldflags: -s -w -X github.com/gocardless/draupnir/version.Version={{.Version}} -X main.commit={{.Commit}}
+  -
+    binary: draupnir-client
+    main: ./cli/main.go
+    ldflags: -s -w -X github.com/gocardless/draupnir/version.Version={{.Version}} -X main.commit={{.Commit}}
+    goos:
+      - linux
+      - darwin
```

Comment:
You've not removed duplicate keys here - you've just indented the array formatting by 1 level.

Reply:
Ran yamllint, which wants the indentation to be this way. The duplicate key was the double `main` that we had for the client

Reply:
oh yeah - totally didn't spot that